examples/postInit/pyrotech.groovy

```diff
@@ -157,13 +192,13 @@ mods.pyrotech.drying_rack.recipeBuilder()
 
 mods.pyrotech.drying_rack.add('apple_to_dirt', item('minecraft:apple'), item('minecraft:dirt'), 1200)
 
-// Kiln:
-// Converts an item into a new one by burning it. Has a chance to fail.
+// groovyscript.wiki.pyrotech.pit_kiln.title:
```

Comment:
since you renamed `Kiln` to `PitKiln`, youll also need to update the lang key name. should also change the title from `Kiln` to `Pit Kiln`, to be consistent

Reply:
It breaks backward compability

Reply:
renaming `Kiln` to `PitKiln` breaks backwards compatability. changing the lang key has no influence on that.
if you want to fix that, add `Kiln` as an alias via modifying the init to include this: `Alias.generateOfClass(PitKiln.class).andGenerate("Kiln")`.

Reply:
Ok

Reply:
still need to rename the lang keys in the lang file, eg like
```diff
-groovyscript.wiki.pyrotech.kiln.title=Kiln
+groovyscript.wiki.pyrotech.pit_kiln.title=Pit Kiln
```